.github/pull_request_template.md

```diff
@@ -0,0 +1,68 @@
+<!--
+### Contribution Checklist
+  
+  - PR title format should be *[type][component] summary*. For details, see *[Guideline - Pulsar PR Naming Convention](https://pulsar.apache.org/contribute/develop-semantic-title/)*. 
+
+  - Fill out the template below to describe the changes contributed by the pull request. That will give reviewers the context they need to do the review.
+  
+  - Each pull request should address only one issue, not mix up code from multiple issues.
+  
+  - Each commit in the pull request has a meaningful commit message
+
+  - Once all items of the checklist are addressed, remove the above text and this checklist, leaving only the filled out template below.
+-->
+
+<!-- Either this PR fixes an issue, -->
+
+Fixes #xyz
+
+<!-- or this PR is one task of an issue -->
+
+Main Issue: #xyz
+
+<!-- If the PR belongs to a PIP, please add the PIP link here -->
+
+### Motivation
+
+<!-- Explain here the context, and why you're making that change. What is the problem you're trying to solve. -->
+
+### Modifications
+
+<!-- Describe the modifications you've done. -->
+
+### Verifying this change
+
+- [ ] Make sure that the change passes the CI checks.
+
+*(Please pick either of the following options)*
+
+This change is a trivial rework / code cleanup without any test coverage.
+
+*(or)*
+
+This change is already covered by existing tests, such as *(please describe tests)*.
+
+*(or)*
+
+This change added tests and can be verified as follows:
+
+*(example:)*
+- *Added integration tests for end-to-end deployment with large payloads (10MB)*
+- *Extended integration test for recovery after broker failure*
+
+### Matching PR in forked repository
+
+PR in forked repository: <!-- ENTER URL HERE -->
+
+<!--
+After opening this PR, the build in apache/pulsar will fail and instructions will
+be provided for opening a PR in the PR author's forked repository.
+
+apache/pulsar pull requests should be first tested in your own fork since the 
+apache/pulsar CI based on GitHub Actions has constrained resources and quota.
+GitHub Actions provides separate quota for pull requests that are executed in 
+a forked repository.
+
+The tests will be run in the forked repository until all PR review comments have
+been handled, the tests pass and the PR is approved by a reviewer.
+-->
```

Comment:
This isn't relevant in this repository.

Reply:
Indeed, the current CI resources of this warehouse are completely sufficient. I will remove this part.